Update for issue #3600 (#3601)
w3f/polkadot-wiki: docs/general/ledger.md

:::

The Polkadot [Ledger][] application is compatible with both the Ledger Nano S and the Ledger Nano X
devices. Ledger devices are hardware wallets that keep your secret key secured on a physical device that

> does not expose it to your computer or the internet. That is, even if you connect your nano via USB to your computer, the private keys will not be leacked. Ledger devices are also hierarchical deterministic wallets (HD wallets), that is:

Reviewer: ```suggestion
does not expose it to your computer or the internet. That is, even if you connect your nano via USB to your computer, the private keys will not be leaked. Ledger devices are also hierarchical deterministic wallets (HD wallets), that is:
```

>   - Deterministic means that there is only one seed phrase to generate all the accounts for different cryptocurrencies.

Reviewer: Not every blockchain token is a cryptocurrency

```suggestion
  - *Deterministic* means that there is only one seed phrase to generate all the accounts for different blockchain networks.
```

>   - Hierarchical means that the accounts are generated in a tree-like structure for different purposes.

Reviewer: ```suggestion
  - *Hierarchical* means that the accounts are generated in a tree-like structure for different purposes.
```

> Ledger devices can be equipped with applications that are blockchain-specific. Such applications are usually develped by third parties and they enable the user to perform essential actions such as transacting and staking.

Reviewer: As we are talking in general, we can keep the wording generic (staking is specific to certain blockchain networks)

```suggestion
Ledger devices can be equipped with applications that are blockchain-specific. Such applications are usually developed by third parties and they enable the user to transact securely on the blockchain network.
```

The Polkadot Ledger application allows you to manage Polkadot's native token, DOT. It supports

> most of the transaction types of the network, including batch transactions from the Utility pallet. Note that the Polkadot Ledger application does not currently support the staking extrinsic `rebag`.

Reviewer: Adding more context and linking to a resource where the reader can find the supported transactions on Polkadot Network. Also, feel free to use https://github.com/Zondax/ledger-polkadot whenever you like to highlight something to the reader.

```suggestion
most of the transaction types of the network, including batch transactions from the Utility pallet. 

:::note Ledger apps may not support all the transactions

Check [Ledger Polkadot App](https://github.com/Zondax/ledger-polkadot) specification for the list of transactions supported. It could be possible that some transactions are supported only on a specific version of the app and some transactions are not supported at all by any version. For instance, the staking `rebag' extrinsic is supported on the XL version but not on the light version. The `setIdentity' extrinsic is not supported by any of the app versions.

:::

```

Here is a list of what you will need before starting:

> - A Ledger Nano X or Nano S plus (reommended since the Polkadot Ledger App takes some space and the Ledger Nano S has limited memory and it is no longer produced).

Reviewer: ```suggestion
- A Ledger Nano X or Nano S plus (recommended for Polkadot Ledger App space requirements. The Ledger Nano S has limited memory and is no longer produced).
```

[![Polkadot on Ledger Live](https://img.youtube.com/vi/jL-N_IWiYVA/0.jpg)](https://www.youtube.com/watch?v=jL-N_IWiYVA)

:::info

> Ledger Live will only show the main account with BIP44 path 44'/354'/0'/0'/0'. This means that if you create a derived account with for example derivation path 44'/354'/0'/0'/1', this will not be displayed on the Ledger Live App. As a consequence it is not possible to transact with derived accounts using the Ledger Live App, but it is possible to do so using Polkadot js. For more information about derived accounts and derivation paths check [the accounts page](../learn/learn-accounts.md).

Reviewer: Good job with linking derivation paths section from accounts page. At this point in the document, the reader is not aware that they can create derived accounts like these on their Ledger devices. Might benefit from some context.

```suggestion
Ledger Live will only show the main account with BIP44 path 44'/354'/0'/0'/0'. This means that if you create a [derived account using Polkadot JS](#using-on-polkadot-js-apps) with a derivation path 44'/354'/0'/0'/1', it will not be displayed on the Ledger Live App. As a consequence it is not possible to transact with derived accounts using the Ledger Live App, but it is possible to do so using Polkadot JS. For more information about derived accounts and derivation paths check [the accounts page](../learn/learn-accounts.md).
```
